### DISCUSSION TITLE
1.20.4崩溃原因看不懂了

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#3083] [提问] [answered]
今天倒腾了一个新版本，崩溃十几次了，但是这次看不懂了
[错误报告-2024-1-6_18.14.07.zip](https://github.com/Hex-Dragon/PCL2/files/13849160/-2024-1-6_18.14.07.zip)

**Comment by burningtnt** [comment, marked as answer]
wits-1.2.1+1.20.3-fabric.jar 这个模组损坏了，重新下载一份吧

**Reply by iPodtouch6557** [reply]
6
<img width="634" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/118656946/569a6fee-5db9-4a86-9d5e-bfd1b64efefb">
测试下来，neoforge是可以下载的，fabric不可以
我也许是误下载了neoforge导致的报错
